napari/components/layerlist.py

```diff
@@ -188,18 +199,22 @@ def _step_size(self) -> np.ndarray:
         """
         return self._get_step_size([layer.extent for layer in self])
 
+    def _step_size_from_scales(self, scales):
```

Comment:
This function is not new code. It was split out from `_get_step_size` for shared use by the new `_ranges` property.